pages/zh-cn/docs/index.mdx

```diff
@@ -31,8 +31,8 @@ labels:
 
 ## ES6 特性
 
-[ES6 部分](/zh-cn/docs/es6/) 描述了三个 ES6 功能组，以及在节点中默认启用的功能细节。 除了注释链接。它还显示如何找到 Node.js 特定版本的 V8 发运。
+[ES6部分](/zh-cn/docs/es6/) 描述了三个ES6功能组，以及在节点中默认启用的功能细节。 除了注释链接。它还显示如何找到Node.js特定版本的V8发运。
```

Comment:
It would be better to translate it as：

```
[ES6 部分](/zh-cn/docs/es6/) 描述了三个 ES6 功能组，
以及在 Node.js 中默认启用的功能细节。除了解释性链接，
它还指示你如何根据特定版本的 Node.js 找到与之对应版本的
V8。
```

Reply:
It's should update that on crowdin

Reply:
Crowdin is right in syntax, but error here.